doc/nrf/ug_nrf9160.rst

```diff
@@ -272,29 +288,29 @@ If you need to restore the original firmware at some point, download the nRF9160
 To program the HEX file, use nrfjprog (which is part of the `nRF Command Line Tools`_).
 
 If you want to route some pins differently from what is done in the preprogrammed firmware, program the :ref:`zephyr:hello_world` sample instead of the preprogrammed firmware.
-Build the sample (located under ``ncs/zephyr/samples/hello_world``) for the nrf9160dk_nrf52840 board.
-To change the routing options, enable or disable the corresponding devicetree nodes for that board as needed.
+Build the sample (located under ``ncs/zephyr/samples/hello_world``) for the nrf9160dk_nrf52840 development kit.
+To change the routing options, enable or disable the corresponding devicetree nodes for that development kit as needed.
 See :ref:`zephyr:nrf9160dk_board_controller_firmware` for detailed information.
 
-Board revisions
-***************
+DK revisions
```

Comment:
This should be Board revisions.

Reply:
fixed, including the following the paragraph